main.tf

```diff
@@ -41,10 +41,12 @@ module "cluster" {
   node_machine_type     = var.node_machine_type
   spot_price            = var.spot_price
   vpc_name              = var.vpc_name
-  vpc_subnets           = var.vpc_subnets
+  public_subnets        = var.public_subnets
+  private_subnets       = var.private_subnets
   vpc_cidr_block        = var.vpc_cidr_block
   force_destroy         = var.force_destroy
   enable_spot_instances = var.enable_spot_instances
+  enable_public_subnet  = var.enable_public_subnet
```

Comment:
I wonder whether it would make sense to rename this variable to clarify it's purpose. AFAIU with this change, both a private and public subnet is created. What this flag determines is into which of these subnets the cluster (and its nodes) get created. What's about `cluster_in_private_subnet` (default true) as a name? Maybe you have a better suggestion?

Reply:
`cluster_in_private_subnet` looks more intuitive. we could default to false and add a note for now as not to break existing installation. then, after some time we can make it true by default.